dlt/destinations/path_utils.py

```diff
@@ -0,0 +1,64 @@
+# this can probably go some other place, but it is shared by destinations, so for now it is here
+from typing import List, Tuple
+
+import pendulum
+import re
+
+from dlt.destinations.exceptions import InvalidFilesystemLayout, CantExtractTablePrefix
+
+# TODO: ensure lauyout only has supported placeholders
+supported_placeholders = {
```

Comment:
maybe this should really go to separate PR? must be really unit tested IMO

Reply:
There are some unit tests, not super many though. I think we can keep it here actually, since the code is not crazy complicated.